projects/kit/components/combo-box/combo-box.component.ts

```diff
@@ -142,15 +142,13 @@ export class TuiComboBoxComponent<T>
         return this.valueContent || this.nativeValue;
     }
 
+    /**
+     * @deprecated
```

Comment:
I don't think it would be a breaking change to remove it now, even if it's a public method. It is used internally only.

Reply:
fixed